pkg/e2e/osd/dedicatedadmin.go

```diff
@@ -265,3 +314,80 @@ func createRolebinding(ns string, user *userv1.User, kind string, kindName strin
 	}, metav1.CreateOptions{})
 	return rb, err
 }
+
+// manageSecrets takes in a list of namespaces
+// and returns error if an action fails
+func manageSecrets(nsList []string, h *helper.H) error {
+
+	for _, ns := range nsList {
+
+		newSecretName := "sample-cust-secret"
+
+		// check 'create' permission
+		dummySecret, err := h.Kube().CoreV1().Secrets(ns).Create(context.TODO(), &corev1.Secret{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      newSecretName,
+				Namespace: ns,
+			},
+		}, metav1.CreateOptions{})
+		Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("failed to create secret %s in namespace %s", newSecretName, ns))
```

Comment:
Instead of _Expect(err).NotTo(HaveOccurred()_, I'd just return the err as line 268 is already making err assertions

Reply:
That wouldn't catch errors across multiple namespaces, so it makes sense (to me) to have an explicit Expect.